book/src/reference/config.md

```diff
@@ -449,6 +449,24 @@ Defaults false (but may become true in the future).
 If you're using GitHub Releases, this will enable GitHub's experimental artifact attestation feature. [See the full docs for details](../supplychain-security/attestations/github.md).
 
 
+### github-custom-job-permissions
+
+> since 0.18.0
+
+Example:
+
+```
+publish-jobs = ["npm", "./my-custom-publish"]
+github-custom-job-permissions = { "my-custom-publish" = ["packages: admin"] }
```

Comment:
I'm kind of torn on the format here. Preformatted yaml-compatible permissions expressions is easier on us to format, but it feels like it introduces new room for user error.